matchzoo/layers/DynamicMaxPooling.py

```diff
@@ -31,16 +43,32 @@ def compute_output_shape(self, input_shape):
         return (None, self.psize1, self.psize2, input_shape_one[3])
 
     @staticmethod
-    def dynamic_pooling_index(len1, len2, max_len1, max_len2):
+    def dynamic_pooling_index(len1, len2, max_len1, max_len2, 
+                              compress_ratio1 = 1, compress_ratio2 = 1):
```

Comment:
This dosen't need to be changed, but for function arguments, we do not need a blank:

```python
def dynamic_pooling_index(len1,
len2,
...
compress_ratio1=1,
compress_ratio2=1
```

Reply:
OK